css/main.css

```diff
@@ -86,22 +94,18 @@ kbd {
 
 
 .topbar{
-	width: 100%;
-	position: fixed;
-	background-color: var(--base-color);
-	top: 0; /* hug the top of the window instead of having a buffer */
-	left: 0;
-	z-index: 1;/*Lets topbar render ontop of iframe*/
-}
-
-.topbarNav{
-	padding: 0.3em;
 	display: flex;
 	flex-wrap: nowrap;
 	justify-content: space-evenly;
 	align-items: stretch;
-	box-sizing: border-box;
+	width: 100%;
 	max-height: 1.8em; /* 2*padding + text height + a teeny bit of padding*/
+	background-color: var(--base-color);
+	padding: 0.3em;
+	box-sizing: border-box;
+	top: 0; /* hug the top of the window instead of having a buffer */
+	left: 0;
+	z-index: 1;/*Lets topbar render ontop of iframe*/
```

Comment:
I don't think we need a z-index here, nothing is overlapping in our layout at this time. Which is what a z-index would resolve.

If we foresee this happening by making something like an overlay, we should keep this, otherwise can we remove it?

Reply:
Ultra also mentioned this, I fixed it. The only thing that needs this z-index value is the flexTopBar so that menu dropdowns can appear on top of the contentContainer.